actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -32,29 +32,22 @@ class InvalidCrossOriginRequest < ActionControllerError #:nodoc:
   # response may be extracted. To prevent this, only XmlHttpRequest (known as XHR or
   # Ajax) requests are allowed to make requests for JavaScript responses.
   #
-  # It's important to remember that XML or JSON requests are also checked by default. If
-  # you're building an API or an SPA you could change forgery protection method in
-  # <tt>ApplicationController</tt> (by default: <tt>:exception</tt>):
+  # Subclasses of <tt>ActionController::Base</tt> are protected by default with the
+  # <tt>:exception</tt> strategy, which raises an
+  # <tt>ActionController::InvalidAuthenticityToken</tt> error on unverified requests.
+  #
+  # It's important to remember that XML or JSON requests are also checked by default.
```

Comment:
I kept this line from the previous version, but reading it again today I don't think it adds anything. Gonna ✂️ it!